modules/network-observability-multitenancy.adoc

```diff
@@ -3,22 +3,31 @@
 // network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="network-observability-multi-tenancy{context}"]
+[id="network-observability-multi-tenancy_{context}"]
 = Enabling multi-tenancy in Network Observability
-Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project administrators. Project administrators who have limited access to some namespaces can access flows for only those namespaces. 
+
+For Developers, multi-tenancy is available for both Loki and Prometheus but requires different access rights.
 
 .Prerequisite
 * You have installed at least link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]
 * You must be logged in as a project administrator
 
 .Procedure
 
-. Authorize reading permission to `user1` by running the following command:
+* To enable limited access to some namespaces for the administrator perspective, authorize reading permission to the `<user_group_or_name>` by running the following command:
```

Comment:
The role `cluster-monitoring-view` does actually give access to cluster-wide metrics, so it is not "limited to some namespaces". I think this would be how it goes:
1. For per-tenant access, users should have the `netobserv-reader` cluster role and the `netobserv-metrics-reader` namespaced role, and use preferably the developer perspective. The admin perspective would work only with Loki.
2. For cluster wide access, users (who are not cluster admins) should have the `netobserv-reader`  and the `cluster-monitoring-view` cluster roles. They can use either the admin perspective or the developer perspective, as they wish.

cc @Amoghrd / @memodi does it sound good to you?

Reply:
(Note that in the second case, the command "oc adm policy add-cluster-role-to-user netobserv-metrics-reader <user_group_or_name>" is only needed until [this bug](https://issues.redhat.com/browse/OCPBUGS-41158) is fixed on the cluster monitoring side)

Reply:
Thanks for the details @jotak . I've updated the documentation accordingly.